flink-connectors/flink-connector-kafka/src/main/java/org/apache/flink/streaming/connectors/kafka/table/KafkaConnectorOptions.java

```diff
@@ -158,6 +158,13 @@
                     .withDescription(
                             "Optional offsets used in case of \"specific-offsets\" startup mode");
 
+    public static final ConfigOption<String> SCAN_END_SPECIFIC_OFFSETS =
```

Comment:
A better name could be `scan.bounded.specific-offsets` to be consistent with the table API